src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+const population = document.querySelectorAll('.population');
+
+function totalSum(numbers) {
+  let sum = 0;
+
+  for (const pop of numbers) {
+    const countryPopulation = pop.textContent.split(',').join('');
+
+    sum += +countryPopulation;
+  }
+
+  return +sum;
```

Comment:
```suggestion
  return Number(sum);
```

Use `Number` for [type casting](https://github.com/mate-academy/style-guides/blob/master/javascript.md#223-numbers-usenumberfor-type-casting-andparseintalways-with-a-radix-for-parsing-strings)